microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365.md

```diff
@@ -133,5 +119,7 @@ Open any remediation item to view details about it, including its name, creation
   - **Failure**: The desired action on remediable emails failed. For example: An admin wants to remove emails from mailboxes, so the admin takes the action of soft-deleting emails. If a remediable email is still found in the mailbox after the action is taken, status will show as failed.
 
   Select any item in the action log to display remediation details. If the details say "successful" or "not found in mailbox," that item was already removed from the mailbox. Sometimes there's a systemic error during remediation. In those cases, it's a good idea to retry remediation.
+  
+  In case of remediating large batches, you can also export the messages send for remediation via Mail Submission and messages which got remediated via Action Logs. The export limit is increased to 100k records.
```

Comment:
Sent rather than send. Also, who is 'you' as in 'you can also'. Is this 'admins'? Is 'Mail Submission' a button or GUI element, or is it mail submission in this context? Also, it's 'messages that were remediated'. The export limit has been increased to 100,000 records (just because this article is translated into something like 36 languages and I'm not sure 100k will be obvious to the translation teams).

Reply:
In case of remediating large batches, admin can also export messages sent for remediation via "Mail Submission" tab and messages that were remediated via "Action Logs" tab. The export limit is increased to 100,000 records.

Reply:
We're good to go!